components/ip5306/ip5306.cpp

```diff
@@ -40,8 +40,7 @@ void IP5306::loop() {
       case 0x80: value = 75; break;
       case 0x00: value = 100; break;
     }
-    if (!this->battery_level_->has_state() || (this->battery_level_->state != value))
-      this->battery_level_->publish_state(value);
+  this->battery_level_->publish_state(value);
```

Comment:
should this be indented a little bit more?

Reply:
Indeed. Fixed it in my fork, but not sure how to 'republish' this in this pull request.